pkg/cmd/cli/describe/describer.go

```
@@ -108,12 +108,24 @@ func (d *BuildDescriber) Describe(namespace, name string) (string, error) {
 		return "", err
 	}
 	events, _ := d.kubeClient.Events(namespace).Search(build)
+	if events == nil {
+		events = &kapi.EventList{}
+	}
+	// get also pod events and merge it all into one list for describe
+	pod, _ := d.kubeClient.Pods(namespace).Get(buildutil.GetBuildPodName(build))
```

You can't eat the error here - that means you'll be passing nil to search.